enterprise/server/scheduling/scheduler_server/scheduler_server.go

```diff
@@ -1815,7 +1816,17 @@ func (s *SchedulerServer) modifyTaskForExperiments(ctx context.Context, executor
 	// with the value in the task.
 	ctx = bazel_request.OverrideRequestMetadata(ctx, taskProto.GetRequestMetadata())
 
-	skipResavingGroup := fp.String(ctx, "skip-resaving-action-snapshots", "", experiments.WithContext("executor_hostname", executorHostname))
+	expOptions := make([]any, 0, 3)
+	expOptions = append(expOptions, experiments.WithContext("executor_hostname", executorHostname))
+	if c, err := claims.ParseClaims(taskProto.GetJwt()); err == nil {
```

Comment:
nit: maybe skip parsing if the JWT is empty, so that if we fail to parse, we know it's something worse than just the JWT being missing, and can log a warning?